app/ui/app.py

```diff
@@ -6,7 +6,7 @@
 import simplejson as json
 from sys import stdout
 
-from gensim.corpora import Dictionary
+from gensim.corpora import Dictionary, MmCorpus
```

Comment:
I want to be on record as saying that I prefer one line per import. This pattern makes it too easy to leave behind unused imports and makes the diffs a bit less readable.

But I don't care enough to actually request that change or stop the merge lol